### DISCUSSION TITLE
MQTT support

### DISCUSSION THREAD

**Discussion by cucpo** [Blackymas/NSPanel_HA_Blueprint#437] [Ideas]
Hi.
It would be great to add MQTT to wake up the display and control individual pages directly from MQTT, something like a wallpanel.

**Comment by hollabee** [comment]
Couldn't you write an automation for this?
Wake up and show individual pages is already possible, see: https://github.com/Blackymas/NSPanel_HA_Blueprint/wiki/(EN)-HowTo---All-important-thing-you-should-know#11-call-certain-pages-directly
